gtsam/geometry/tests/testPose2.cpp

```diff
@@ -835,6 +835,139 @@ TEST(Pose2 , ChartDerivatives) {
   CHECK_CHART_DERIVATIVES(T2,T1);
 }
 
+//******************************************************************************
+namespace transform_covariance3 {
+  const double degree = M_PI / 180;
+
+  template<class TPose>
+  static typename TPose::Jacobian generate_full_covariance(
```

Comment:
naming, pls use Google style guide: `GenerateFullCovariance` (capitalized since static). Also, add at least a one-line comment? Goes for all functions below...

Reply:
Changed names and added function comments.